.changeset/metal-waves-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-site": patch
+---
+
+Prevent the admin's block-preview from becoming unresponsive when rendering an `input` in the frontend
```

Comment:
I'd either use a generic message

```suggestion
Prevent infinite recalculation of preview overlays in the block preview
```

or remove the term "frontend", since both Admin and site are technically frontends, which can lead to confusion.

```suggestion
Prevent the block-preview from becoming unresponsive when rendering an `input`
```

Reply:
Good point, I like the second one